File: syncloud/src/main/java/org/syncloud/android/core/redirect/model/Domain.java
package org.syncloud.android.core.redirect.model;

import java.io.Serializable;
import java.util.List;

public class Domain implements Serializable {
    public String user_domain;
    public String device_mac_address;
    public String device_name;
    public String device_title;
    public Boolean map_local_address;
    public String web_protocol;
    public Integer web_local_port;
    public Integer web_port;
    public String ip;
    public String local_ip;
    public String last_update;
}

File: syncloud/src/main/java/org/syncloud/android/core/redirect/model/Domain.kt
package org.syncloud.android.core.redirect.model

import org.syncloud.android.core.platform.model.DomainModel
import java.io.Serializable

data class Domain(
    val name: String,
    val device_name : String,
    var device_title: String,
    var map_local_address: Boolean,
    var web_protocol: String,
    var web_local_port: Int,
    var web_port: Int
) : Serializable

fun List<Domain>.toModels(): List<DomainModel> {
    return this.map { DomainModel(it) }
}